src/PlatformSockets.c

```diff
@@ -10,14 +10,15 @@ void addrToUrlSafeString(struct sockaddr_storage* addr, char* string)
 
     if (addr->ss_family == AF_INET6) {
 #ifdef __vita__
-        LC_ASSERT(FALSE);
+        struct sockaddr_in* sin = (struct sockaddr_in*)addr;
+        inet_ntop(addr->ss_family, &sin->sin_addr, addrstr, sizeof(addrstr));
```

Comment:
In PlatformSockets.h, you've already defined sockaddr_in6 to sockaddr_in and sin6_port to sin_port. I think if you also define sin6_addr to sin_addr, you can remove this `#ifdef`